app/src/main/res/layout/welcome1.xml

```diff
@@ -1,20 +1,69 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/scrollView"
```

Comment:
Similar comment to above: Can we please preface/namespace these so they start with `welcome_` or `welcome_wizard_`? It helps when your IDE is suggesting completions to pick the right one.

Reply:
removed unnecessary id.